src/composer.ts

```diff
@@ -48,44 +65,15 @@ type MountMap = {
     }
   }
 
+interface GameQueryUpdate extends tt.Update.CallbackQueryUpdate {
+  callback_query: tt.CallbackQuery.GameShortGameCallbackQuery
+}
```

Comment:
Add to `typegram`?

Reply:
I don't think it belongs there, or else we should also add `TextMessageUpdate` and the like. That would introduce a lot of redundancy.

`telegraf` also just needs it because we explicitly support subtypes of _message_, but nothing else. As a consequence, we need this helper interface when we want to work with other types of `Update`s that contain unions.